src/index.rst

```diff
@@ -31,6 +31,19 @@ The following resources are less focused on summarizing the academic literature
  * `The Whole Code Catalog <https://futureofcoding.org/catalog/>`_
 
 
+. toctree::
+
+.. raw:: latex
+
+  \part{Interactive Theorem Prover}
+
+.. toctree::
+  :caption: Interactive Theorem Prover
```

Comment:
this should go in the Live Programming chapter, not in a separate top-level chapter like this